ArduCopter/mode.cpp

```diff
@@ -258,7 +258,7 @@ bool Copter::set_mode(Mode::Number mode, ModeReason reason)
 
 #if MODE_ZIGZAG_ENABLED == ENABLED && SPRAYER_ENABLED == ENABLED
```

Comment:
... and this should be in `exit_mode`, preferably in the zigzag cpp file... future....